### DISCUSSION TITLE
Welcome to freeRASP for the Cordova community!

### DISCUSSION THREAD

**Discussion by msikyna** [talsec/Free-RASP-Cordova#6] [Announcements]
## 👋 Welcome to freeRASP for the Cordova community!
  
Here you can find only Cordova-specific topics.

All the freeRASP platform agnostic topics will be migrated to freeRASP [Community Discussion](https://github.com/talsec/Free-RASP-Community/discussions) here: 

https://github.com/talsec/Free-RASP-Community/discussions

**Comment by meeranaik** [comment]
Hi,
Is this plugin compatible with cordova-android version 7.1.2
